pkgs/development/tools/pip-tools.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, pythonPackages }:
+buildPythonPackage rec {
```

Comment:
Actually, it did belong there because, while it is an application, users need it for the specific environment they're using it with. Therefore it has to be in `python-packages.nix`. It's the same as the `pip` package or `pytest`.